test/e2e/e2e_test.go

```diff
@@ -114,7 +114,7 @@ var _ = ginkgo.Describe("E2E Test Suite", func() {
 		testsuites.InitGcsFuseCSIMetadataPrefetchTestSuite,
 	}
 
-	testDriver := specs.InitGCSFuseCSITestDriver(c, m, *bucketLocation, *skipGcpSaTest, false)
+	testDriver := specs.InitGCSFuseCSITestDriver(c, m, *bucketLocation, *skipGcpSaTest, false, false)
```

Comment:
so if we want to run a separate prow job  with gRPC enabled, this bool would need to become a flag, and that would be setup in the run-e2e*.sh is my understanding